src/pages/HomePage.js

```diff
@@ -1,44 +1,79 @@
 import React from 'react';
-import {authProvider} from '../providers/LoginUser'
+import { authProvider } from '../providers/LoginUser'
 import { useUser } from '../providers/UserContextProvider'
-import { Spinner } from 'react-bootstrap'
-import { Link } from 'react-router-dom';
+import { Spinner, Container, Navbar, Nav, Jumbotron, Card } from 'react-bootstrap'
+import { House, Person } from 'react-bootstrap-icons'
 
-export const HomePage = ()=>{
-    const { dispatch } = useUser()
+export const HomePage = () => {
+    const { state, dispatch } = useUser()
```

Comment:
Besides logging the user state, are we using the state anywhere else? If not, you can remove the state from this file. The user state will be propagated across the application through the context at the root. Other than this, the rest of the files look good.